[2023-11-14T11:03:52.999+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: extract_dag.extract_task manual__2023-11-14T11:03:50.745719+00:00 [queued]>
[2023-11-14T11:03:53.004+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: extract_dag.extract_task manual__2023-11-14T11:03:50.745719+00:00 [queued]>
[2023-11-14T11:03:53.004+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T11:03:53.004+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-14T11:03:53.004+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T11:03:53.050+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): extract_task> on 2023-11-14 11:03:50.745719+00:00
[2023-11-14T11:03:53.053+0000] {standard_task_runner.py:55} INFO - Started process 15617 to run task
[2023-11-14T11:03:53.056+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'extract_dag', 'extract_task', 'manual__2023-11-14T11:03:50.745719+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/extract_dag.py', '--cfg-path', '/tmp/tmpb8hvqbbd']
[2023-11-14T11:03:53.057+0000] {standard_task_runner.py:83} INFO - Job 6: Subtask extract_task
[2023-11-14T11:03:53.125+0000] {task_command.py:388} INFO - Running <TaskInstance: extract_dag.extract_task manual__2023-11-14T11:03:50.745719+00:00 [running]> on host codespaces-1a8ad4
[2023-11-14T11:03:53.237+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=extract_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2023-11-14T11:03:50.745719+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-14T11:03:50.745719+00:00
[2023-11-14T11:03:53.239+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-11-14T11:03:53.239+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wget -c https://datahub.io/core/top-level-domain-names/r/top-level-domain-names.csv.csv -O             workspaces/hands-on-introduction-data-engineering-4395021/lab/orchestrated/airflow-extract-data.csv']
[2023-11-14T11:03:53.249+0000] {subprocess.py:86} INFO - Output:
[2023-11-14T11:03:53.254+0000] {subprocess.py:93} INFO - workspaces/hands-on-introduction-data-engineering-4395021/lab/orchestrated/airflow-extract-data.csv: No such file or directory
[2023-11-14T11:03:53.255+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-11-14T11:03:53.264+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/python/3.10.8/lib/python3.10/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-11-14T11:03:53.266+0000] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=extract_dag, task_id=extract_task, execution_date=20231114T110350, start_date=20231114T110352, end_date=20231114T110353
[2023-11-14T11:03:53.329+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 6 for task extract_task (Bash command failed. The command returned a non-zero exit code 1.; 15617)
[2023-11-14T11:03:53.347+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-11-14T11:03:53.360+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
